packages/sources/coinmarketcap/src/endpoint/crypto.ts

```diff
@@ -9,7 +9,7 @@ import {
 } from '@chainlink/types'
 
 export const supportedEndpoints = ['crypto', 'price', 'marketcap']
-export const batchablePropertyPath = ['base', 'convert']
+export const batchablePropertyPath = [{ name: 'base' }, { name: 'convert' }]
```

Comment:
Since we know the limit here let's add it:
```suggestion
export const batchablePropertyPath = [{ name: 'base' }, { name: 'convert', limit: 120 }]
```